compiler-rt/include/profile/InstrProfData.inc

```diff
@@ -702,6 +708,8 @@ serializeValueProfDataFrom(ValueProfRecordClosure *Closure,
 #define INSTR_PROF_COVMAP_COMMON __llvm_covmap
 #define INSTR_PROF_COVFUN_COMMON __llvm_covfun
 #define INSTR_PROF_ORDERFILE_COMMON __llvm_orderfile
```

Comment:
This just a nit, but could you move `orderfile` here and elsewhere to the bottom so that all `__llvm_cov*` entries are next to each other for easier readability.

Reply:
Done.